plugins/parsers/dropwizard/parser.go

```diff
@@ -160,14 +175,19 @@ func (p *Parser) unmarshalMetrics(buf []byte) (map[string]interface{}, error) {
 	return jsonOut, nil
 }
 
-func (p *Parser) readDWMetrics(metricType string, dwms interface{}, metrics []telegraf.Metric, tm time.Time) []telegraf.Metric {
+func (p *Parser) readDWMetrics(metricType string, dwms interface{}, metrics []telegraf.Metric, tm time.Time) ([]telegraf.Metric, error) {
 	if dwmsTyped, ok := dwms.(map[string]interface{}); ok {
 		for dwmName, dwmFields := range dwmsTyped {
 			measurementName := dwmName
 			tags := make(map[string]string)
 			fieldPrefix := ""
 			if p.templateEngine != nil {
-				measurementName, tags, fieldPrefix, _ = p.templateEngine.Apply(dwmName)
+				//nolint:errcheck // Potential error not worth propagating
```

Comment:
This needs to be removed, doesn't it?

Reply:
Good catch! I had to exclude the unused nolint:errcheck checks temporarily since there were already some in excluded directories, so soon this should be caught automatically again 😄